templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -30,15 +30,13 @@ contents:
            exit 1
         fi
       else
-        GOOGLE_URL=https://storage.googleapis.com/etcd
-        DOWNLOAD_URL=${GOOGLE_URL}
-
-        echo "Downloading etcdctl binary.."
-        curl -s -L ${DOWNLOAD_URL}/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -o $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz \
-          && tar -xzf $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -C $ASSET_DIR/shared --strip-components=1 \
-          && mv $ASSET_DIR/shared/etcdctl $ASSET_DIR/bin \
-          && rm $ASSET_DIR/shared/etcd \
-          && $ETCDCTL_BIN version
+        local etcdimg="{{.Images.etcdKey}}"
+        local etcdctr=$(podman create ${etcdimg})
+        local etcdmnt=$(podman mount "${etcdctr}")
+        cp ${etcdmnt}/bin/etcdctl $ASSET_DIR/bin
+        umount "${etcdmnt}"
+        podman rm "${etcdctr}"
+        $ETCDCTL_BIN version
```

Comment:
I am not sure, why it had worked in the past, but to me, `version` without dashes is giving an error.
[core@ip-10-0-136-108 ~]$ ./assets/bin/etcdctl version
No help topic for 'version'
[core@ip-10-0-136-108 ~]$ ./assets/bin/etcdctl --version
etcdctl version: 3.3.17
API version: 2